cmake/Dependencies.cmake

```diff
@@ -1821,18 +1822,6 @@ list(APPEND Caffe2_DEPENDENCY_LIBS fmt::fmt-header-only)
 set(BUILD_SHARED_LIBS ${TEMP_BUILD_SHARED_LIBS} CACHE BOOL "Build shared libs" FORCE)
 
 # ---[ Kineto
-if(USE_KINETO AND INTERN_BUILD_MOBILE)
-  message(STATUS "Not using libkineto in a mobile build.")
```

Comment:
nit: the message is confusing - it seems like we are expanding to Android/iOS which are "mobile". is `INTERN_BUILD_MOBILE` for FB internal mobile builds?

Reply:
i'm removing this, we're expanding to all builds yes